oobi/benchmark-definitions/python/data_types.py

```diff
@@ -75,9 +96,7 @@ class Model(object):
   name: str
   # Tags that describe the model characteristics.
   tags: List[str]
-  framework_type: ModelFrameworkType
-  # Source of the model implementation.
-  source_info: str
+  meta_model_id: str
```

Comment:
Should this be `meta_model: MetaModel`?

Reply:
I wasn't sure whether to use the id or the object. Using the object means one less lookup. I'll update to use the object.